core/pkg/opencost/allocation.go

```diff
@@ -2587,13 +2676,13 @@ func (a *Allocation) SanitizeNaN() {
 		log.DedupedWarningf(5, "Allocation: Unexpected NaN found for GPUHours name:%s, window:%s, properties:%s", a.Name, a.Window.String(), a.Properties.String())
 		a.GPUHours = 0
 	}
-	if math.IsNaN(a.GPURequestAverage) {
-		log.DedupedWarningf(5, "Allocation: Unexpected NaN found for GPURequestAverage name:%s, window:%s, properties:%s", a.Name, a.Window.String(), a.Properties.String())
-		a.GPURequestAverage = 0
+	if math.IsNaN(a.DeprecatedGPURequestAverage) {
+		log.DedupedWarningf(5, "Allocation: Unexpected NaN found for DeprecatedGPURequestAverage name:%s, window:%s, properties:%s", a.Name, a.Window.String(), a.Properties.String())
+		a.DeprecatedGPURequestAverage = 0
 	}
-	if math.IsNaN(a.GPUUsageAverage) {
-		log.DedupedWarningf(5, "Allocation: Unexpected NaN found for GPUUsageAverage name:%s, window:%s, properties:%s", a.Name, a.Window.String(), a.Properties.String())
-		a.GPUUsageAverage = 0
+	if math.IsNaN(a.DeprecatedGPUUsageAverage) {
+		log.DedupedWarningf(5, "Allocation: Unexpected NaN found for DeprecatedGPUUsageAverage name:%s, window:%s, properties:%s", a.Name, a.Window.String(), a.Properties.String())
+		a.DeprecatedGPUUsageAverage = 0
```

Comment:
Don't think we care anymore about the deprecated fields. Safe to remove.

Reply:
Fixed